core/trino-main/src/test/java/io/trino/operator/scalar/TestJsonExtract.java

```diff
@@ -329,6 +330,13 @@ public void testInvalidExtracts()
         assertInvalidExtract("{ \"store\": { \"book\": [{ \"title\": \"title\" }] } }", "$.store.book[", "Invalid JSON path: '$.store.book['");
     }
 
+    @Test
```

Comment:
Is this test failing if disable changing limits?

Reply:
```
java.io.UncheckedIOException: com.fasterxml.jackson.core.exc.StreamConstraintsException: String length (20000001) exceeds the maximum length (20000000)

	at io.trino.operator.scalar.JsonExtract.extract(JsonExtract.java:144)
	at io.trino.operator.scalar.TestJsonExtract.doJsonExtract(TestJsonExtract.java:368)
	at io.trino.operator.scalar.TestJsonExtract.testExtractLongString(TestJsonExtract.java:337)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	...
```